pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -854,6 +893,65 @@ LogicalResult ConvertAtenOp<AtenDropoutOp>::matchAndRewrite(
   return success();
 }
 
+// Ref: https://pytorch.org/docs/stable/generated/torch.Tensor.expand.html
+// aten.broadcast_to has similar semantics with torch.expand
+template <>
+LogicalResult ConvertAtenOp<AtenBroadcastToOp>::matchAndRewrite(
+    AtenBroadcastToOp op,
+    OpAdaptor adaptor,
+    ConversionPatternRewriter& rewriter) const {
+  // Not a tensor type.
+  //
+  auto selfType = adaptor.self().getType().dyn_cast<TensorType>();
+  if (!selfType)
+    return op.emitError("Only tensor types are currently supported");
+
+  SmallVector<Value> dimsSize;
```

Comment:
nit: `_` connected variable name?

Reply:
Indeed, LLVM standards prefer camel casing for the variables.

Reply:
oops ... they even starts with upper case ... https://github.com/llvm/llvm-project/blob/169417531578501e955f12c79ecb8ff05333ae15/llvm/lib/TableGen/StringMatcher.cpp#L135